src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let amount = 40 * days;
+
```

Comment:
p2

Reply:
I can`t use here const variable, cause it changes in the if